CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 2.3.0 (Unreleased)
 
+## 2.2.1 (Febuary 14, 2020)
+BUG FIXES
+- Resolves a panic and several type issues with the ASN field.
```

Comment:
Don't some IP whitelist fixes also land in this version?

Reply:
There were some ip whitelist fixes included in 2.2.0, not sure if that's what you are referring to?

Reply:
D'oh that was it. Forgot that made it into 2.2.0